robot/client/client_test.go

```diff
@@ -1061,6 +1061,13 @@ func TestClientDialerOption(t *testing.T) {
 }
 
 func TestClientResources(t *testing.T) {
+	// TODO(RSDK-6185): robot.proto importing app.proto causes this test to fail
+	// (gRPC reflection breaks for the RobotService proto service because the RDK
+	// does not know how to generate Go code for tagger.proto since that
+	// generation requires a protoc-gen-gotag Golang dep that the RDK does not
+	// have).
+	t.Skip()
```

Comment:
See my TODO comment above. I broke this test and gRPC reflection for `robot.proto` by trying to import `app.proto` in `robot.proto` 😮‍💨 . It should be fixed with https://github.com/viamrobotics/api/pull/429, but until then, we should probably skip this test, as it will fail in any RDK PR attempting to upgrade the version of the `go.viam.com/api` dependency.

Reply:
Yeah that would be cool 🤔 . We tried to do this in the Mongo Go driver, too, but found it was better to assert that things failed in a particular way (i.e. a test that passes when certain things fail) rather than ensure that `go test` fails a certain `t.Run`.